lib/livebook/intellisense.ex

```diff
@@ -401,16 +401,16 @@ defmodule Livebook.Intellisense do
 
     # Don't add module prefix to operator signatures
     if :binary.match(signatures_string, ["(", "/"]) != :nomatch do
-      module_to_prefix(module) <> signatures_string
+      module_to_string(module) <> "." <> signatures_string
     else
       signatures_string
     end
   end
 
-  defp module_to_prefix(mod) do
+  defp module_to_string(mod) do
```

Comment:
Isn't this function equivalent to calling `inspect(mod)` now? :)

Reply:
Smart! :cat: